streams/src/main/java/org/apache/kafka/streams/processor/internals/PartitionGroup.java

```diff
@@ -207,11 +207,11 @@ boolean allPartitionsBuffered() {
     public void close() {
         clear();
         partitionQueues.clear();
+        streamTime = RecordQueue.UNKNOWN;
```

Comment:
@mjsax this is to skip resetting the partition time so that we do not need to re-initialize metadata upon resuming again.